pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -913,7 +916,7 @@ func (adc *attachDetachController) GetNodeName() types.NodeName {
 }
 
 func (adc *attachDetachController) GetEventRecorder() record.EventRecorder {
-	return adc.recorder
+	return nil
```

Comment:
this seems to break some assumptions

Reply:
This was never set anywhere - so it's supposed to be a no-op change :)

[As you see - I removed `recorder` field completely and didn't have to change any code for it.]

Reply:
yeah, it's is weird, is just to meet some Interface or something?, the `GetNodeName()` above is returning empty too

```
grep -r GetEventRecorder pkg/controller/
pkg/controller/volume/attachdetach/attach_detach_controller.go:func (adc *attachDetachController) GetEventRecorder() record.EventRecorder {
pkg/controller/volume/expand/expand_controller.go:func (expc *expandController) GetEventRecorder() record.EventRecorder {
pkg/controller/volume/persistentvolume/volume_host.go:func (ctrl *PersistentVolumeController) GetEventRecorder() record.EventRecorder {
```